prow/tide/tide.go

```diff
@@ -1286,7 +1349,7 @@ type subpool struct {
 	pjs []prowapi.ProwJob
 	prs []PullRequest
 
-	cc contextChecker
+	cc map[int]contextChecker
 	// presubmit contains all required presubmits for each PR
 	// in this subpool
 	presubmits map[int][]config.Presubmit
```

Comment:
Why does this still exist and who uses it, if we're dynamically resolving the list in `accumulateBatch`?

Reply:
It is being used in:

* `filterPR` to filter out PRs with failing contexts or contexts that are not controlled by Prow
* `accumulate` to determine the current state of PRs